iroh-gateway/src/core.rs

```diff
@@ -48,9 +66,12 @@ impl Core {
     }
 
     pub async fn serve(self) {
+        // todo(arqu): ?uri=... https://github.com/ipfs/go-ipfs/pull/7802
         let app = Router::new()
             .route("/ipfs/:cid", get(get_ipfs))
             .route("/ipfs/:cid/*cpath", get(get_ipfs))
+            .route("/ipfs/ipfs/:cid", get(redundant_ipfs))
```

Comment:
ugh, I am guessing you added these because go-ipfs supports them? do we really want this, seems like folks could just like not type ipfs twice 😅

Reply:
Kinda, it was part of our flow doc too so I just replicated it. There's better ways to possibly handle it, but unnecessary until we support more types. I guess, they have it because it was enough of an issue, maybe lots of bad linking or redirects in general, or just for dev UX?

Happy to drop it as it is a bit of an eye sore.

Reply:
lets keep and make a note of it in the flow doc that we might not want to do this cc @b5